.circleci/config.yml

```diff
@@ -542,8 +552,14 @@ jobs:
     <<: *windows-defaults
     steps:
       - checkout
-      - run: $NodeVersion = Get-Content -Path .\.nvmrc; nvm install $NodeVersion; nvm use $NodeVersion
-      - run: npm install --global yarn
+      - run:
+          name: Setup Node.js and Yarn
+          command: |
+            choco uninstall nvm.portable
```

Comment:
There is an issue with backslashes in paths with the NPM package with the Node v18 version using NVM. The Chocolately version seems to be working fine.